pkg/dss/geo/s2.go

```diff
@@ -120,24 +157,34 @@ func loopAreaKm2(loop *s2.Loop) float64 {
 // Covering calculates the S2 covering of a set of S2 points. Will try the loop
 // in both clockwise and counter clockwise.
 func Covering(points []s2.Point) (s2.CellUnion, error) {
-	loop := s2.LoopFromPoints(points)
-	if loopAreaKm2(loop) <= maxAllowedAreaKm2 {
-		return RegionCoverer.Covering(loop), nil
+	cu, err := CoveringForLoop(s2.LoopFromPoints(points))
+	switch err.(type) {
+	case nil:
+		return cu, nil
+	case *ErrAreaTooLarge:
+		// Empty on purpose.
```

Comment:
Let's make this clearer; something like "Area may be too large because vertices were wound in the opposite direction; check below"

Reply:
cross-check: You mean that we should log the fact that we are retrying with reversed order?

Reply:
Nah, just the comment (for developer readability, why is an error handler empty?)

Reply:
Done